common/app/model/meta.scala

```diff
@@ -344,7 +342,6 @@ final case class MetaData(
     sensitive: Boolean = false,
     isFoundation: Boolean = false,
     firstPublicationDate: Option[DateTime] = None,
-    isFacebookHttpsTest: Boolean = false,
```

Comment:
Deleting a field which might have been pressed by the Fronts presser 🤔 
I'm going to let the presser run in CODE overnight then try this branch but I suspect Scala JSON is  might gracefully ignore extra files.

Reply:
This seems ok.
Deployed main to CODE and let it press the uk sport front.
Deployed this branch and the front still rendered in CODE.